Userland/Libraries/LibWeb/Painting/BorderPainting.cpp

```diff
@@ -137,7 +138,66 @@ void paint_border(PaintContext& context, BorderEdge edge, Gfx::IntRect const& re
         return;
     }
 
-    context.painter().fill_rect(rect, color);
+    auto draw_horizontal_or_vertical_line = [&](auto p1, auto p2) {
+        // Note: Using fill_rect() here since draw_line() produces some overlapping pixels
+        // at the end of a line, which cause issues on borders with transparency.
+        p2.translate_by(1, 1);
+        context.painter().fill_rect(Gfx::IntRect::from_two_points(p1, p2), color);
+    };
+
+    auto draw_border = [&](auto const& border, auto const& radius, auto const& opposite_border, auto const& opposite_radius, auto p1_step_translate, auto p2_step_translate) {
+        auto current_p1 = p1.to_type<float>();
+        auto current_p2 = p2.to_type<float>();
```

Comment:
CI's not happy about this

Reply:
Yeah, I'm very confused about that. Apparently you can't capture the results of a structured binding in clang??? :confounded: